yogstation/code/game/world.dm

```diff
@@ -32,3 +32,43 @@ GLOBAL_LIST_EMPTY(donators)
 		P = GLOB.preferences_datums[ckey]
 		if(P)
 			P.unlock_content |= 2
+
+/world/update_status()
+
+	//BASIC SHIT
+	var/s = ""
+	var/server_name = CONFIG_GET(string/servername)
+	if (server_name)
+		s += "<b>[server_name]</b> &#8212; "
+	
+	s += "<b>[station_name()]</b>]<br>"; // The station & server name line
+	s += "(<a href=\"https://forums.yogstation.net/index.php\">Forums</a>|<a href=\"https://discord.gg/8hphvMe\">Discord</a>)<br>" // The Forum & Discord links line
+	s += "Mode: <b>[GLOB.master_mode]</b><br>" // The Gamemode line
```

Comment:
You should probably leave this line out if it's secret since it's almost always secret.

Reply:
I thought about doing exactly that, but, I wanted to maximize the number of lines we were using, for better ad space.